journal-letters/modernized/01-george-fox-testimony.adoc

```diff
@@ -31,5 +31,10 @@ And as for his opposers, they withered and vanished away;
 and after he had finished his testimony, he died in the Lord, and is blessed,
 and rests from his labors, and his works follow him.
 
---George FoxGooses, +++[+++near Brentwood, Essex, his son-in-law William Mead`'s house,
+[.signed-section-signature]
+--George Fox
```

Comment:
I don't think I have this in the styleguide anywhere, (maybe I should add it), but we remove any dashes before signatures, so this should just be:

```ASCIIDOC
[.signed-section-signature]
George Fox
```